Remotion/Web/Core/Res/Themes/NovaViso/HTML/DropDownMenu.css

```diff
@@ -1,374 +1,207 @@
-  color: Black;
-  outline: none;
+  background-repeat: no-repeat;
 }
 
 ul.DropDownMenuOptions li.selected a
 {
   color: Black;
-  border: solid 1px #7CB2E6;
-  background-color: #E5F0FC;
-  padding: 0 2px 2px 0;
-}
-
-ul.DropDownMenuOptions li.selected a:active,
-ul.DropDownMenuOptions li a:active
-{
-  color: Black;
-  border: solid 1px #569CE0;
-  background-color: #C5E0FC;
-  padding: 0 2px 2px 0;
-}
-
-/* Workaround for IE-issue with a span inside an anchor tag blocking the :active pseudo class. */
-body.msie ul.DropDownMenuOptions li a
-{
-  position: relative;
-}
-
-/* Workaround for IE-issue with a span inside an anchor tag blocking the :active pseudo class. */
-body.msie ul.DropDownMenuOptions li a::before
-{
-  content: "";
-  position: absolute;
-  left: 0;
-  right: 0;
-  top: 0;
-  bottom: 0;
-}
-
-ul.DropDownMenuOptions li a img
-{
-  border: 0;
-  margin-right: 15px;
-  vertical-align: middle;
-}
-
-ul.DropDownMenuOptions li a span
-{
-  display: inline-block;
-  padding: 2px 0px;
+  background-color: var(--color-highlight);
+  border-radius: var(--remotion-themed-border-radius);
 }
 
 ul.DropDownMenuOptions li.DropDownMenuItemDisabled a
 {
+  display: flex;
   cursor: default;
   border-style: none;
-  padding: 1px 3px 3px 3px;
-  color: gray;
-}
-
-ul.DropDownMenuOptions li.DropDownMenuItemDisabled.selected a
-{
-  background-color: #ECEEEC;
+  color: var(--color-contrast-separator);
 }
 
-ul.DropDownMenuOptions li.DropDownMenuItemDisabled a:active
+span.DropDownMenuContainer span.disabled a.DropDownMenuLabel
 {
-  background-color: transparent;
-  border-style: none;
-  padding: 1px 3px 3px 3px;
-  color: gray;
+  color: var(--color-contrast-separator);
 }
 
+ul.DropDownMenuOptions li.DropDownMenuItemDisabled.selected a,
 ul.DropDownMenuOptions li.DropDownMenuItemDisabled.selected a:active
 {
-  background-color: #ECEEEC;
+  background-color: var(--color-standard-separator);
 }
 
-/* style separator */
-span.DropDownMenuSeparator
-{
-  display: block;
-  line-height: 1px;
-  margin-left: 30px;
-  border: 0;
-  background-color: #AAAAAA;
-}
-
-span.DropDownMenuItemIcon
-{
-  margin-right: 15px;
-  width: 16px;
-  padding: 0px;
-}
-
-div.DropDownMenuStatus
-{
-  overflow: hidden;
-  z-index: 2147483647;
-  /* position and dimension are set via javascript */
-  max-height: 4em;
-  max-width: 30em;
-}
-
-div.DropDownMenuStatus > div
-{
-  width: 100%;
-  padding: 0.25em;
-  border: 1px solid Black;
-  background-color: Lightyellow;
-  box-sizing: border-box;
-  -moz-box-sizing: border-box;
-  -webkit-box-sizing: border-box;
-}
-
-div.DropDownMenuStatus.loading > div
-{
-  background-image: url('../Image/sprite.svg#DropDownMenuRequestInProgressIndicator');
-  background-size: 16px 16px;
-  background-repeat: no-repeat;
-  background-position: 0.25em;
-  padding-left: calc(16px + 0.25em);
-  min-height: 20px;
-}
-
-div.DropDownMenuStatus.error > div
+ul.DropDownMenuOptions li.DropDownMenuItemDisabled a:active
```

Comment:
please separte the dropdown-otpions from the dropdownmenu-button. right now, they're interleaved.

Reply:
To be completely honest, I'm not entirely sure what is meant here specifically. I couldn't find any interleaving of the dropdown options and the dropdown button itself. Separating `.DropDownMenuOptions` and `.DropDownMenuItemDisabled`, as the comment referred to that line, did not work, though.

Reply:
When you check the file, you will notice that there's declarations for `ul.DropDownMenuOptions`, delcarations for `span.DropDownMenuSelect` and declarations for other elements. They are not grouped into separate sections within the file but they should be to make things easy to read and understand.